_i18n/en.yml

```diff
@@ -103,13 +107,4 @@ en:
       - September
       - October
       - November
-      - December
-
-press:
-  mentions: Press mentions
-  inquiry: For Press inquiries contact us at
-
-resources_title: Resources
-resources:
```

Comment:
@acao this line again caused some indent issued, I'm wondering if it's not expecting nested anonymous collections? Anyhow, removing this, fixed the issue

Reply:
I will take a look!